opts/network.go

```diff
@@ -0,0 +1,100 @@
+package opts
```

Comment:
We will need to move the opts/* to docker/cli. I believe that should be a seperate PR. I am going to park this here until then.